### ISSUE TITLE
merlin-locate-in-new-window is ignored after #1426

### ISSUE THREAD

**Issue by lthms** [ocaml/merlin#1460]
I’ve been using `merlin-locate-in-new-window` set to `'never` for a while to prevent Merlin to open a new window.

Recently, it stopped working altogether, and it as far as I can tell, it is related to https://github.com/ocaml/merlin/pull/1426.

I am not familiar enough with prefix argument, but the fact that this change breaks an existing configuration was a bit… annoying :sweat_smile:. Fortunately, I’ve been able to recover my previous setup by defining my own helper around `merlin-locate`:

```elisp
(defun +merlin-locate ()
    (interactive)
    ;; force `merlin-locate-in-new-window' to be 'never
    (merlin-locate '(4)))
```

I have a fairly customized Emacs configuration, but just calling `M-x merlin-locate` ignores `merlin-locate-in-new-window`, wihch looks like a bad UX to me.

**Comment by trefis** [comment]
Ping @panglesd

**Comment by panglesd** [comment]
Indeed, it is a bug introduced by my PR, fixed by #1461. Sorry about that!
